ui/app/components/clients/page/overview.hbs

```diff
@@ -4,6 +4,7 @@
 ~}}
 
 <Clients::RunningTotal
+  @isSecretsSyncActivated={{@isSecretsSyncActivated}}
```

Comment:
you'll notice i'm passing `isSecretsSyncActivated` allllll the way down from the `clients/counts` route to a bunch of different components. i did this instead of calling `this.version.hasSecretsSync` because i assume we don't want to show all that messaging if the feature isn't activated/in use yet? 

on second thought...oh no.... we probably do want to show secret sync things even if the feature isn't activated yet in `RunningTotal` and `Attribution` ...but maybe not `UsageCounts` 🤦

Reply:
Answered in slack, but following up here: 

We want to show anythings sync related if the user has the feature and if it is activated. It's possible customers may **never** activate the feature. (Product can decide later if they think showing 0 secret syncs in this instances is useful). Therefore, we only need to display sync data if the feature is activated, which will give an accurate `0` count if secrets haven't been synced yet.